src/Core/Batch/BatchConfig.php

```diff
@@ -0,0 +1,122 @@
+<?php
+/**
+ * Copyright 2017 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Cloud\Core\Batch;
+
+/**
+ * Hold configurations for the \Google\Cloud\Core\Batch\BatchRunner.
+ */
+class BatchConfig
+{
+    /**
+     * @var \Google\Cloud\Core\Batch\BatchJob[]
```

Comment:
In the same namespace, there's no need to use the FQCN. `BatchJob[]` is sufficient.